access_utils/general_stats_parse.cwl

```diff
@@ -0,0 +1,49 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+id: general_stats_parse
+label: general_stats_parse
+baseCommand: general_stats_parse.py
+inputs:
+  - id: dir
+    type: Directory
+    inputBinding:
+      position: 0
+      prefix: '--dir'
+    doc: Directory containing results.
+  - id: samples-json
+    type: File
+    inputBinding:
+      position: 0
+      prefix: '--samples-json'
+    doc: Sample JSON file.
+outputs:
+  - id: sample_meta_tumor
+    type: File
```

Comment:
I would make both outputs optional. We may run the tool with only plasma or only buffy coat samples